docs/adminguide/updating/index.rst

```diff
@@ -0,0 +1,49 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+.. _doc_adminguide_updating:
+
+.. cspell:words linenos literalinclude
+
+======================
+Parsec Server updating
+======================
+
+This guide covers the updating procedure for the `Parsec`_ server (also called `parsec-backend`).
+
+.. note::
+
+  We consider that you are using the same infrastructure configuration as in the :ref:`Parsec Server installation guide <doc_adminguide_hosting>`.
+  Adapt this guide to your own situation.
+
+.. _Parsec: https://parsec.cloud
+
+Updating to a newer version
+***************************
+
+.. warning::
+
+  Before updating to a newer version, we recommend that you backup the database,
+  In case you need to rollback the update.
+
+In this guide, we will consider that you are using the parsec-backend at the version ``2023-10-11-v2.16.0-rc.5.dev-b33d909`` and you want to update-it to ``v2.16.0``.
+
+1. Update the docker image's tag of the ``parsec-backend`` in docker-compose's file:
+
+  .. literalinclude:: ../hosting/parsec-server.docker.yaml
+    :language: yaml
+    :linenos:
+    :emphasize-lines: 43
+
+  You need to change the used tag (at line **43**) to ``v2.16.0``.
+
+2. Apply the database migration:
+
+  .. code-block:: bash
+
+    docker-compose compose -f parsec-server.docker.yaml run parsec-backend migrate
```

Comment:
What about the dry-run that we apply before migrating the database? shouldn't we suggest the user to do the same thing?

Reply:
We can ask the user to run `dry-run` first, but then what does it do with that information ?

The user will likely don't know what to do with that information.

Reply:
The instructions in `parsec-cloud-saas-deploy` shows an example run:

```
$ parsec backend migrate --db $DATABASE_URL --dry-run
Migrate ✔
0001_initial.sql (already applied)
0002_add_migration_table.sql (already applied)
0003_human_handle.sql (already applied)
0004_invite.sql (already applied)
0005_redacted_certificates.sql (already applied)
0006_outsider_enabled.sql (already applied)
0007_users_limit.sql (already applied)
0008_apvi1_removal.sql (already applied)
0009_add_realm_user_change_table.sql (already applied)
0010_add_pki_certificate_table.sql (already applied)
0011_add_sequester_tables.sql (already applied)
```

And then say :
> Verify that the changes listed by the command are expected.

It's still not that clear, maybe we can say something like:   
```
Check that SQL migration scripts are executed incrementally with the status "already applied". 
If there is an error, please open a bug report so we can investigate.
```

BTW, can you add @vxgmichel as a reviewer?

Reply:
done